requirements.txt

```diff
@@ -6,4 +6,5 @@ flask
 todoist_api_python
 requests
 pyright
-sqlalchemy
\ No newline at end of file
+sqlalchemy
+pydantic
```

Comment:
should we add a trailing newline to allow for future diffs to not pick up the last line